frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card-item.tsx

```diff
@@ -92,6 +92,15 @@ export const BucketsItem: React.FC<BucketsItemProps> = React.memo(
   ),
 );
 
+export const getTitleComponent = (objectsCount: string, link: string) => ({ children }) => (
```

Comment:
I'd just make this a regular component
```suggestion
export const BucketsTitleComponent: React.FC<...> = ({objectsCount, link, children}) => (
```

and use React.useCallback where you need to pass the `objectsCount` and `link`